core/src/test/java/com/linecorp/armeria/common/stream/StreamMessageDuplicatorTest.java

```diff
@@ -324,6 +324,7 @@ public void lastDuplicateStream() {
         }
         for (int i = 25; i < 30; i++) {  // rest of them are still in the queue.
             assertThat(bufs[i].refCnt()).isOne();
+            bufs[i].release();
```

Comment:
Do you think we should instead `await` on the refcnts after `close`? I think `close` is supposed to release these, but it might be asynchronuos

Reply:
Yeah, I was going to fix in that way :-)